lib/releaselib.sh

```diff
@@ -815,6 +815,7 @@ release::docker::release () {
   local binaries=(
```

Comment:
https://github.com/kubernetes/release/pull/355 and https://github.com/kubernetes/kubernetes/pull/47939.